internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -86,9 +86,9 @@ func NewApplicationSnapshotImage(ctx context.Context, image string, publicKey st
 		checkOpts: checkOpts,
 	}
 
-	k8s, err := kubernetesCreator()
+	k8s, err := kubernetesClientCreator()
```

Comment:
Why don't we just call `kubernetes.NewClient` directly? Easier to mock in tests?

Reply:
We need this in tests, though that test was removed in a recent refactor. I'm working on bringing it back in.